### ISSUE TITLE
Question: How can we display Pipeline Analytics on dashboards

### ISSUE THREAD

**Issue by awentzel** [MicrosoftDocs/vsts-docs#5201]
What methods are available today to consume pipeline analytics in Microsoft Teams as a Tab, Power BI, Embedded in Website, or Sharepoint?

Thanks,
Aaron

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2da4714e-d931-6777-d8be-b123a2b0fa17
* Version Independent ID: 09eec061-51ab-e6a9-e61c-0b6f618cd969
* Content: [Pipeline reports - Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/reports/pipelinereport?view=azure-devops)
* Content Source: [docs/pipelines/reports/pipelinereport.md](https://github.com/MicrosoftDocs/vsts-docs/blob/master/docs/pipelines/reports/pipelinereport.md)
* Product: **devops**
* Technology: **devops-cicd**
* GitHub Login: @Additi
* Microsoft Alias: **admahesh**

**Comment by Additi** [comment]
Hi Aaron, we have our OData APIs, which you can use to pull data in Power BI and can even build your own extension in MS teams to consume. 
Link: https://docs.microsoft.com/en-us/azure/devops/report/extend-analytics/analytics-metadata?view=azure-devops
Thanks,
Aditi

**Comment by WilliamAntonRohm** [comment]
@awentzel -- Aaron, thank you for your question. I am closing this issue now.  You are welcome to @ mention me for any followup.
We hope to hear from you again.

**Comment by wadechandler** [comment]
@Additi @WilliamAntonRohm  I know this has been closed, but this question is relevant to a full view from this issue and related documentation as I can't seem to find it. I have used the odata endpoint to query data. However, how can I use a named analytics view from that odata endpoint without making a new query? I figured the metadata would have some views area to show where they are and how to access them, but this isn't the case, and there is no mention of those views in the linked documentation.
